--- a/programs/cmdline_options.c
+++ b/programs/cmdline_options.c
@@ -62,15 +62,15 @@

 #include "cmdline_options.h"

-// #ifndef _MSC_VER
-// static inline int strcat_s(char* restrict dest, size_t destsz, const char* restrict src)
-// {
-// 	if (dest == 0 || src == 0)
-// 	{
-// 		return -1;
-// 	}
-// 	strcat(dest, src);
-// 	return 0;
-// }
-// #endif
+#ifndef _MSC_VER
+static inline int strcat_s(char* restrict dest, size_t destsz, const char* restrict src)
+{
+	if (dest == 0 || src == 0)
+	{
+		return -1;
+	}
+	strcat(dest, src);
+	return 0;
+}
+#endif

